tutorials/users/ide/ide.md

```diff
@@ -1 +1,9 @@
 # IDE
+
+Jupyter is a web-based development environment well suited for arranging workflows in 
```

Comment:
I'd move the last sentence first... talk about Marble in the first sentence, introduce the concept of an IDE, and _then_ discuss jupyter

Reply:
Did another write-up.  Not sure if I should keep the part in the IDE explanation  about "source code editor,
build automation tools, and a debugger."